File: FisioAppAPI/DTOs/PhysiotherapistProfileDto.cs
namespace FisioAppAPI.DTOs;

public class PhysiotherapistProfileDto
{
    public Guid Id { get; set; }
    public Guid UserId { get; set; }
    public string LicenseNumber { get; set; } = null!;
    public string LicenseAuthority { get; set; } = null!;
    public string Specialties { get; set; } = null!; // JSON: "Traumatología|Deportiva|Neurológica"
    public int GraduationYear { get; set; }
    public bool IsActive { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
}

File: FisioAppFront-Angular/src/app/shared/dtos/physiotherapist-profile.dto.ts
export interface PhysiotherapistProfileDto {
  id: string;
  userId: string;
  licenseNumber: string;
  licenseAuthority: string;
  specialties: string; // JSON: "Traumatología|Deportiva|Neurológica"
  graduationYear: number;
  isActive: boolean;
  createdAt: Date;
  updatedAt: Date;
}
